piccolo/apps/schema/commands/generate.py

```diff
@@ -107,31 +108,25 @@ def is_foreign_key(self, column_name: str) -> bool:
                 return True
         return False
 
-    def get_foreign_key_constraint_name(self, column_name) -> str:
+    def get_foreign_key_constraint_name(
+        self, column_name
+    ) -> t.Tuple[str, str] | ValueError:  # type: ignore
```

Comment:
We can't use the new `|` syntax for union unfortunately , as I don't think there's a back port to Python versions < 3.10.

I don't think exceptions can be put in type annotations - it would be nice if it did though. I think the only way is to put it in the docstring:

```python
:raises: ValueError
```

Reply:
Yes, you are right. I forgot about that. But it's a cool thing though. I'll change it.
Ah, I didn't know that about exceptions. thank you.